packages/models-library/setup.cfg

```diff
@@ -0,0 +1,25 @@
+[bumpversion]
+current_version = 0.1.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
+
+[bumpversion:file:src/models_library/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
+
+[bdist_wheel]
+universal = 1
+
+[flake8]
```

Comment:
we do not really use flake, right?

Reply:
well no idea where this is coming from... copy paste probably.. let me fix this.

Reply:
coming from service library??

Reply:
anyway it's removed